slave.mk

```diff
@@ -574,17 +575,17 @@ $(addprefix $(TARGET_PATH)/, $(SONIC_INSTALLERS)) : $(TARGET_PATH)/% : \
         onie-image.conf \
         build_debian.sh \
         build_image.sh \
-        $$(addsuffix -install,$$(addprefix $(STRETCH_DEBS_PATH)/,$$($$*_DEPENDS))) \
-        $$(addprefix $(STRETCH_DEBS_PATH)/,$$($$*_INSTALLS)) \
-        $$(addprefix $(STRETCH_DEBS_PATH)/,$$($$*_LAZY_INSTALLS)) \
+        $$(addsuffix -install,$$(addprefix $(STRETCH_DEBS_PATH)/,$$($$(subst $(SUFFIX_DBG_IMAGE_MARK),$(NULL),$$*)_DEPENDS))) \
+        $$(addprefix $(STRETCH_DEBS_PATH)/,$$($$(subst $(SUFFIX_DBG_IMAGE_MARK),$(NULL),$$*)_INSTALLS)) \
```

Comment:
>$(subst $(SUFFIX_DBG_IMAGE_MARK),$(NULL),$$*) [](start = 47, length = 45)

this has been repeatedly used in many places, can you create a variable to define it and use it afterwards.

Reply:
All "subst ..." are dropped as it become redundant by not having to change the target name, per later comment